BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeDeepLinkActivity.kt

```diff
@@ -7,11 +7,10 @@ import androidx.appcompat.app.AppCompatActivity
  * This Activity is used by the SDK Activity to handle deep links on behalf of the merchant
  * It should not be used directly.
  */
+// NEXT_MAJOR_VERSION : remove this class
 class BraintreeDeepLinkActivity : AppCompatActivity() {
-    private val browserSwitchClient = BrowserSwitchClient()
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        browserSwitchClient.captureResult(this)
```

Comment:
This is unrelated, but the latest browser switch snapshot removes this method so it no longer compiles